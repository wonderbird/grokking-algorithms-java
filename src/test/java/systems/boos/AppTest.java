package systems.boos;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

class AppTest {

  @Test
  void executeDemos_approvalTest() {
    Output output = new Output();
    App.executeDemos(output);
    Approvals.verify(output);
  }
}
