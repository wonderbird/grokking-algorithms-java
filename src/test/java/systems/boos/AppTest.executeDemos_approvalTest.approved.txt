Binary Search
=============
Search 9 -> 8 is the index; 9 is the value in [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

Selection Sort
==============

Shuffled input: [2, 4, 5, 1, 10, 9, 7, 3, 8, 6]
   Sorted list: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

